anova_tutorial.Rmd

```diff
@@ -0,0 +1,284 @@
+# ANOVA tutorial
+
+Xinrui Bai and Zewen Shi
+
+
+```{r, echo=FALSE}
+knitr::opts_chunk$set(error = TRUE)
+```
+
+The main **motivation** for this community contribution project is to provide a tutorial of a few useful visualization tools in the process of the analysis of variances, a common topic that statisticians deal with on a daily basis. This project will walk users through such tools of visualization and how they would be a complement to concrete test results by making them understandable even by audiences with little training in statistics. In the first part, the background for a typical problem of our choice will be provided and manipulations of the data will be carried out to obtain a model of best fit for later use. In the second part, diagnosis using visualization tools will be conducted and results will be analyzed.
+
+```{r}
+library(car)
+library(MASS)
+library(tidyverse)
+library(glmnet)
+library(ggplot2)
+library(ggpubr)
+library(tidyverse)
+library(broom)
+#library(AICcmodavg)
+
+```
+
+
+## Part 1: Background and Preliminary Data Manipulation
+
+
+**Analysis of variance (ANOVA)** is a collection of statistical models and their associated estimation procedures (such as the "variation" among and between groups) used to analyse the differences among means. ANOVA was developed by the statistician Ronald Fisher. ANOVA is based on the law of total variance, where the observed variance in a particular variable is partitioned into components attributable to different sources of variation. In its simplest form, ANOVA provides a statistical test of whether two or more population means are equal, and therefore generalizes the t-test beyond two means.(Wikipedia)
+
+
+### 1.Taking a first look at the dataset: 
```

Comment:
Other hashtag headers should not be followed by numbers since the hashtags will create numbered subheadings. Correct: ## Subheading. Incorrect: ## 3. Subheading.